### DISCUSSION TITLE
Who has the password for `executablebooks@gmail.com`?

### DISCUSSION THREAD

**Discussion by choldgraf** [executablebooks/meta#412] [Q&A] [answered]
In our docs we recommend that people email `executablebooks@gmail.com` but it's not clear who has access to this account, or who has the password for it. I looked through my records but I can't find it...maybe somebody else set it up? (cc @jstac @mmcky @chrisjsewell @AakashGfude ?)

**Comment by chrisjsewell** [comment, marked as answer]
Me, I control everything 😉
